src/scene-entry-manager.js

```diff
@@ -75,6 +75,8 @@ export default class SceneEntryManager {
       return;
     }
 
+    this.scene.setAttribute("motion-capture-replayer", "enabled", false);
```

Comment:
Should we remove the component entirely?

Reply:
I believe its used for bots